### PR TITLE
Was export PDFWStreamForBuffer missed in the definition?

### PR DESCRIPTION
**Pull request by jakefreeberg**
I had to add this to my project to get hummus replaced with muhammara. I hope this helps your repo. Thank you for carrying the project forward.

### CODE REVIEW

**Comment by julianhille** [comment]
Please add a changelog entry then we're good to go.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,60 +7,66 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.7.1] - 2021-05-10
```
Comment (julianhille) [inline CHANGELOG.md]: Please remove this.
I'll add a date and version number as soon as I release

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,60 +7,66 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
 ## [1.7.0] - 2021-03-08
 
 ### Added
 
-* More electron 11.x releases
-* Add electron 12.0.0
-* Add electron 10.2, 10.3
-* Add electron 9.4, 9.3
+- More electron 11.x releases
```
Comment (julianhille) [inline CHANGELOG.md]: Please revert this changes and below. 
if you'd like to lint this file open up a second pr and add a changelog entry.

Thank you for your time.

**Review by julianhille — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jakefreeberg) [reply CHANGELOG.md]: Sorry... my VSCode did more than I'd realized and still left a bit of format mess, but nothing seems to have visibly changed so hopefully we're good now.

**Review by julianhille — APPROVED** [review]
Please squash and then i merge. 

Thank you for your contribution

**Comment by jakefreeberg** [comment]
Thank you for dealing with my mess here. I don't contribute to open source nearly as much as I should, but learned how to do a better job next time I do.

**Pull request merged**